src/pygama/flow/file_db.py

```diff
@@ -200,7 +200,7 @@ def set_config(self, config: dict, config_path: str = None) -> None:
         # Relative paths are interpreted relative to the configuration file
         if not data_dir.startswith("/"):
             config_dir = os.path.dirname(config_path)
-            data_dir = os.path.join(config_dir, data_dir)
+            data_dir = os.path.join(config_dir, data_dir.lstrip("/"))
```

Comment:
this may fail if a user doesn't end an entry in the fdb_config.json with a '/' -- better to leave it off and let the user remove it in the json